src/portal/src/lib/components/list-replication-rule/list-replication-rule.component.ts

```diff
@@ -158,6 +157,35 @@ export class ListReplicationRuleComponent implements OnInit, OnChanges {
         ) {
             this.deleteOpe(message.data);
         }
+        if ( message &&
```

Comment:
else if ?

Reply:
should not use "else if", they are independent